<center><h1>Parametric Gaussian VaR</h1></center>

It assumes portfolio returns follow a specific probability distribution (usually normal distribution) and estimates VaR based on the mean and standard deviation of returns.

```z``` value for a ```VaR``` (5%) = -1.65 

Daily Standard deviation (```σ```) = 0.014 

Portfolio value = 1000000
	

```VaR```	= ```z * σ```

```VaR``` (5%) on ```INR``` basis = ```VaR * Portfolio value``` 

---

#### VaR

In [2]:
z = -1.65
stdev = 0.014

In [3]:
var = z * stdev

In [4]:
var

-0.0231

#### VaR (5%) on INR basis

In [5]:
pftv = 1000000

In [6]:
var * pftv

-23100.0

### Concusion
There is a ```5%``` probabaility that on any given day, the loss in value on this particular asset will equal or exceed ```2.31%``` or ```INR 23,100```

---

# If the expected return is non zero
For a INR 1000000 portfolio, the expected 1-week portfolio return and standard deviation are 0.00188 and .0125 respectively. Calculate the 1 week VaR at 5% level of significance.

```VaR``` = ```[Expected return-zσ] * portfolio value```	

```Expected return``` = 0.00188

```Standard deviation (σ)``` = 0.0125

```z value for a VaR (5%)``` = -1.65

```Portfolio value``` = 1000000

---

In [11]:
expected_return = 0.00188
stddev= 0.0125
z_value = -1.65
Pftv = 1000000

```VaR``` = (```Expected_return``` - (```Standard_deviation ``` * ```z_value```))*```Portfolio_value```

In [12]:
VaR = (expected_return - (stddev * z_value)) * Pftv

In [13]:
VaR

22505.0

```VaR``` = 22505.0

The banker can be 95% confident that the maximum 1-week loss will not exceed INR 22505	

---
